aten/src/ATen/core/stack.h

```diff
@@ -103,9 +103,7 @@ inline void drop(Stack* stack, size_t n) {
   drop(*stack, n);
 }
 inline IValue pop(Stack& stack) {
-  if (stack.empty()) {
-    throw std::runtime_error("pop() called on empty stack");
-  }
+  TORCH_CHECK(not stack.empty(), "pop() called on empty stack");
```

Comment:
```suggestion
  TORCH_CHECK(!stack.empty(), "pop() called on empty stack");
```

Reply:
@malfet This is unfortunately valid in C++17...

Reply:
@Skylion007 , well, according to VC++2019 it's `not true` (pun intended) :)